bin/npm.cmd

```diff
@@ -1,17 +1,6 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-:: Try to find git.exe in path
```

Comment:
Are there not scenarios where atom or a user might fire of this npm.cmd
without going via apm.cmd?

On Sun, Jul 17, 2016, 3:53 PM Nathan Sobo <notifications@github.com> wrote:

> In bin/npm.cmd <https://github.com/atom/apm/pull/592#discussion_r71091478>
> :
>
> > @@ -1,17 +1,6 @@
> >  @echo off
> >  setlocal EnableDelayedExpansion
> >
> > -:: Try to find git.exe in path
>
> We didn't understand why it was needed as part of npm.cmd... Made more
> sense in apm.cmd. Assumed it was copied and pasted. Am I wrong?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/atom/apm/pull/592/files/e104233655433cea85ed9af20e900b7e1c4b644b#r71091478>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHQp5VKPkaiZ91OIljfXm_peGOU7gpoks5qWrKCgaJpZM4JOD1L>
> .
>